lib/msal-browser/docs/login-user.md

```diff
@@ -124,6 +128,24 @@ try {
 }
 ```
 
+### Without User Hint
+
+If there is not enough information available about the user, you can attempt to use the `ssoSilent` API **without** passing an `account`, `sid` or `login_hint`.
+
+```javascript
+const silentRequest = {
+    scopes: ["User.Read", "Mail.Read"]
+};
+```
+
+However, be aware that if your application has code paths for multiple users in a single browser session, or if the user has multiple accounts for that single browser session, then there is a higher likelihood of silent sign-in errors. You may see the following error show up in the event of multiple account sessions found by the authorization server:
+
+```txt
+InteractionRequiredAuthError: interaction_required: AADSTS16000: Either multiple user identities are available for the current request or selected account is not supported for the scenario.
+```
+
+This indicates that the server could not determine which account to sign into, and will require either one of the parameters above (`account`, `login_hint`, `sid`) or an interactive sign-in to choose the account.
```

Comment:
We need to make this really clear, and should probably show a code sample for how to handle this.